maintenance_request_ifs_check/models/maintenance.py

```diff
@@ -0,0 +1,40 @@
+from odoo import api, models, fields, _
+from odoo.exceptions import ValidationError
+
+
+class MaintenanceStage(models.Model):
+    _inherit = 'maintenance.stage'
+
+    ifs_checked = fields.Boolean(
+        'IFS checked',
+        default=False)
+
+
+class MaintenanceRequest(models.Model):
+    _inherit = 'maintenance.request'
+
+    ifs_check_repaired_correctly = fields.Boolean(
+        string='Equipment repaired correctly')
+    ifs_check_free_strange_elements = fields.Boolean(
+        string='Free strange elements')
+    ifs_check_collected_location = fields.Boolean(
+        string='Collected location')
+    ifs_check_cleaning_area = fields.Boolean(
+        string='Cleaning the area')
+    ifs_checked = fields.Boolean(
+        string='IFS checked',
+        related='stage_id.ifs_checked',
+        store=True,
+        readonly=True)
```

Comment:
I'm not sure if `stored` and `readonly` attributes are needed. Is `store=True` needed for searching support? Also, `readonly=True` is the default starting from v12.

Reply:
Ok. I'll remove store and readonly. I checked that store=True is not necessary for searching support.